Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+
+          // These define keys that are always "CONTAINED" since they typically are a combination of key values
+          // stored as separate and/or multiple attributed values that may be combined as in the case of flags
+          // and masks.  There are some that have NO direct cross reference (i.e. NSSupport, NSBGColor, etc)
+          // Each of the ones listed here will MOST PROBABLY have an entry in the 'XmlKeyToDecoderSelectorMap'
+          // below that provides a cross referenced to an asociated decoding method...
+          // If there is an easy way to check whether an existing OLD XIB key is contained within the XIB 5
+          // version the 'containsValueForKey:' method in this file should be modified and the key omitted from this
+          // list (i.e. NSContents, NSAlternateContents, NSIntercellSpacingWidth, NSIntercellSpacingHeight, etc)...
+          XmlKeysDefined = [NSArray arrayWithObjects: @"NSWindowBacking", @"NSWTFlags",
+                                    @"NSvFlags", @"NSBGColor",
+                                    @"NSSize", //@"IBIsSystemFont",
+                                    //@"NSHeaderClipView",
+                                    @"NSHScroller", @"NSVScroller", @"NSsFlags", @"NSsFlags2",
+                                    @"NSColumnAutoresizingStyle", @"NSTvFlags", @"NScvFlags",
+                                    @"NSSupport", @"NSName",
+                                    @"NSMenuItem",
+                                    @"NSDocView",
+                                    @"NSSliderType",
+                                    @"NSCellPrototype", @"NSBrFlags", @"NSNumberOfVisibleColumns",
+                                    @"NSWhite", @"NSRGB", @"NSCYMK",
+                                    //@"NSContents", @"NSAlternateContents",
+                                    @"NSCellFlags", @"NSCellFlags2",
+                                    @"NSButtonFlags", @"NSButtonFlags2",
+                                    @"NSSelectedIndex", @"NSUsesItemFromMenu",
+                                    @"NSNormalImage", @"NSAlternateImage",
+                                    @"NSBorderType", @"NSBoxType", @"NSTitlePosition",
+                                    @"NSTitleCell", @"NSOffsets",
+                                    @"NSMatrixFlags", @"NSNumCols", @"NSNumRows",
+                                    @"NSSharedData", @"NSFlags", @"NSTVFlags", @"NSDefaultParagraphStyle",
+                                    @"NSpiFlags", nil];
+          RETAIN(XmlKeysDefined);
+
+          // These define XML tags (i.e. '<autoresizingMask ...') to an associated decode method...
+          XmlTagToDecoderSelectorMap = [NSDictionary dictionaryWithObjectsAndKeys:
+                                                       @"decodeTableColumnResizingMaskForElement:", @"tableColumnResizingMask",
+                                                       //@"decodeAutoresizingMaskForElement:", @"autoresizingMask",
```

Comment:
If unnecessary, let's remove this. If possibly necessary in the future, let's clarify why it's commented out.

Reply:
Done